server/back-end/models/wordmodel.js

```diff
@@ -0,0 +1,52 @@
+const sql = require("./db.js");
+const { response } = require("express");
+
+// Word
+const Word = function (word) {
+    this.word_name = word.name;
+    this.created_at = Date.now()// add new timestamp ;
+    this.updated_at = Date.now()// add new timestamp ;
+    this.use_yn = word.use_yn;
```

Comment:
Can you add a comment as to what use_yn is?

Reply:
use_yn is to determine if we're using the word or not (yes or no). Think of it as a boolean

Reply:
Can you give me more context on this use_yn? Why and how is it being used?